### PR TITLE
Document that we're moving on from this project

### PR DESCRIPTION
**Pull request by AsaAyers**
@swang Before making this official, I wanted to run it past you. 

The comment markers are something I saw [eslint does](https://raw.githubusercontent.com/eslint/eslint/master/.github/ISSUE_TEMPLATE.md). Even if you leave that text it won't show up in the issue or PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,14 +10,22 @@ For guides on installing, using and configuring CoffeeLint, head over
...
 
-## Team
+## :rotating_light: Team :rotating_light:
+
+There are no future plans for CoffeeLint. Only major bugs will be addressed
+here. If you need a rule, write and publish it on npm. See [the documentation on
+how to build your own
+rules.](https://github.com/clutchski/coffeelint/blob/master/doc/integration.md)
+
+[You Don't Need CoffeeScript. -Asa](https://gist.github.com/AsaAyers/2e6cb944753b667d5f90ae1c954607cb)
```
Comment (adam-lynch) [inline README.md]: 404

File: README.md [hunk 2]
```diff
@@ -10,14 +10,22 @@ For guides on installing, using and configuring CoffeeLint, head over
 To suggest a feature, report a bug, or general discussion, head over
 [here](http://github.com/clutchski/coffeelint/issues/).
 
-## Team
+## :rotating_light: Team :rotating_light:
+
+There are no future plans for CoffeeLint. Only major bugs will be addressed
```
Comment (adam-lynch) [inline README.md]: I assume you're willing to add maintainers if they volunteered though? (Or transfer ownership or whatever). Maybe that should be mentioned here?

File: README.md (hunk 1 above)
Comment (AsaAyers) [inline README.md]: oops, that was a draft version I deleted.

File: README.md (hunk 2 above)
Comment (AsaAyers) [inline README.md]: I don't think I want to put out a call for a new maintainer. I've seen people come and go over the years and advancing this project is actually a lot of work. I'd want to see how they intend to advance the project before trying to hand anything over though. If they're passionate enough about the project I'm assuming they'll ignore the warning and contact me about it.

I actually don't have full control of the project. @clutchski gave me commit access after I made a series of PRs a few years ago. If someone wants commit access, we still need to contact him about it.

Comment (adam-lynch) [inline README.md]: 👍 fair enough

Comment (clutchski) [inline README.md]: i think at this point people should just fork it if they care. none of us have used coffeescript in years. i think it's fine to stick a 🍴 in this one.

**Comment by swang** [comment]
Sorry caught a fever over the weekend. Still bit sick now.

It looks like CoffeeScript is prepping for a new release soon (a year+ since the last version), and it seems like Jeremy is back from his vacation? I'm guessing as soon as that gets released there will probably be some bugs to fix. I'd be interested in working on this if they ever get back to releasing regularly to get on par with es2015 features.

I think additionally we should provide a list/accept PRs for the third party rules that want to get added. That way it's less of a blackhole for users.

I've been pretty busy with work but (assuming a new version gets released soon), I'd like to delve in and see if any of the problems we've had before (indentation, difference in blank lines, undef/unused vars) can be fixed up.

**Comment by AsaAyers** [comment]
There's no rush. I'm just trying to publicly document what's already been happening here for probably a year.

Do you want to abandon this PR and you become sole maintainer? If you're going to continue, that's fine, but I think I'll unsubscribe from the repo so I'm only notified of mentions. Whether CoffeeScript makes a new release or not, I don't intend to continue this project. CoffeeScript is fundamentally broken and provides very few benefits over just using standard JavaScript https://gist.github.com/AsaAyers/d09e4de118b8d6b5e2d8fa3e38e496e0

Do you want to update the branch for this PR to better reflect your plans?

**Comment by lydell** [comment]
> It looks like CoffeeScript is prepping for a new release soon

See #580.

**Comment by AsaAyers** [comment]
I'm now unsubscribed from this repo because I don't want to know about new issues being opened. 👋

**Pull request closed without merging**